test/fixtures/annotate/dynamic-extends/output7.js

```diff
@@ -0,0 +1,9 @@
+const base = bar => class {
+  foo() {
+    return bar;
+  }
+};
+
+class MyClass extends
+/*#__PURE__*/
+base() {}
```

Comment:
I guess guessed that this would be the output expected by the test suite. But I can't reproduce this formatting with a fresh install of the dependencies here (due to no lock file) :-)

Reply:
Well, this repo could use some work on setting up github actions instead of Travis, updating deps and just dropping Babel 6 support 😅